kernel/os/src/arch/pic32/asm/ctx.S

```diff
@@ -90,20 +88,24 @@
     sw	    k0, CTX_EPC(sp)
     mfc0    k0, _CP0_BADVADDR
 	sw	    k0, CTX_BADVADDR(sp)
+	mfc0    k0, _CP0_CAUSE
+	sw      k0, CTX_CAUSE(sp)
+
```

Comment:
These instructions are not aligned, or tabs maybe...

Reply:
tabs removed